kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java

```diff
@@ -248,4 +283,34 @@ int runInit() {
     }
     return executionResult;
   }
+
+  static class KyuubiBeelineResourceBundle extends ListResourceBundle {
+    static String CMD_USAGE = "cmd-usage";
+
+    private Object[][] contents = new Object[beelineResourceBundle.keySet().size()][];
+
+    public KyuubiBeelineResourceBundle() {
+      int i = 0;
+      for (String key : beelineResourceBundle.keySet()) {
+        String value = beelineResourceBundle.getString(key);
+        if (key.equals(CMD_USAGE)) {
+          StringBuilder stringBuilder = new StringBuilder();
+          stringBuilder.append(value).append("\n");
+          stringBuilder
+              .append("Usage: java " + KyuubiBeeLine.class.getCanonicalName())
+              .append("\n");
+          stringBuilder.append(
+              "   --python-mode=[true/false]                      Execute python code/script.");
```

Comment:
Maybe add tips to the migration guide `docs/deployment/migration-guide.md`

Reply:
done